Makefile

```diff
@@ -90,6 +91,12 @@ else
 CARGO_OSDK_ARGS += --boot-method="$(BOOT_METHOD)"
 endif
 
+ifeq ($(COVERAGE), 1)
+FEATURES += coverage
+RUSTFLAGS += -Cinstrument-coverage -Zno-profiler-runtime
+CARGO_OSDK_ARGS += --qemu-args="--no-shutdown" # Keep QEMU running for coverage report
```

Comment:
Coverage support should be an OSDK feature. Enabling the right combination of Cargo features, Rust flags, and QEMU flags should be taken care of by OSDK. Could you propose the CLI interface for OSDK's the coverage support?